tests/monotouch-test/SceneKit/GeometrySourceTest.cs

```diff
@@ -37,11 +41,16 @@ public class GeometrySourceTest {
 		[Test]
 		public void FromMetalBuffer ()
 		{
+#if MONOMAC
+			if (!TestRuntime.CheckMacSystemVersion (10, 11))
```

Comment:
Same: replace `CheckSystemVersion` with call to `TestRuntime.CheckXcodeVersion`.